tests/test_TFNetworkLayer.py

```diff
@@ -3065,6 +3065,266 @@ def mask_func(source, **kwargs):
       last_var_v = var_v
 
 
+def _create_data(n_batch, n_time, n_feature, network, dtype=tf.int32, offset=0, sparse=True):
```

Comment:
Note that there is `make_feed_dict` in that function, which has some similar logic. Maybe it makes more sense to reuse that, and not duplicating the logic.